src/wallet_new.ts

```diff
@@ -94,10 +94,12 @@ export class WalletDAO {
 		arAmount: number,
 		fromWallet: Wallet,
 		toAddress: ArweaveAddress,
-		appName = 'ArDrive-Core',
-		appVersion = '1.0',
-		trxType = 'transfer',
-		otherTags?: GQLTagInterface[]
+		[
+			{ value: appName = 'ArDrive-Core' },
+			{ value: appVersion = '1.0' },
+			{ value: trxType = '1.0' },
```

Comment:
default value should be `transfer` rather than `1.0`

Reply:
Here: e8cd2d4c8277f5bbb0fc5f22bdd5ffe3a003885e